packages/frameworks/test/frameworks.unit.test.ts

```diff
@@ -246,3 +291,16 @@ describe('frameworks', () => {
     );
   });
 });
+
+function getValidator() {
+  const ajv = new Ajv();
+
+  ajv.addKeyword('isFunction', {
+    compile: shouldMatch => data => {
+      const matches = data instanceof Function;
```

Comment:
I'd prefer `typeof data === 'function'`, but not a huge deal.

Reply:
I like that better as well. [`bed27ff` (#9900)](https://github.com/vercel/vercel/pull/9900/commits/bed27ffd21c2cd7de6783452f9491b7241f34dc0)